src/languages/julia-repl.js

```diff
@@ -4,6 +4,7 @@ Description: Julia REPL sessions
 Author: Morten Piibeleht <morten.piibeleht@gmail.com>
 Website: https://julialang.org
 Requires: julia.js
+Category: system
```

Comment:
Does system just mean "compiled to machine code"?

Reply:
That's kinda what I was going when I was categorizing these. However, I've corrected Julia to be scientific since that's what it's mostly used for.